website/content/installation/_index.md

```diff
@@ -112,3 +112,10 @@ secretGenerator:
 generatorOptions:
  disableNameSuffixHash: true
 ```
+
+## Upgrade Current Version
+To upgrade current version of metallb to new release, always look at [the release notes](https://metallb.universe.tf/release-notes/) to see the changes and action required items. Pay special attention when upgrading major/minor releases.
+
+After that we can install new metallb version by downloading the new manifests and `kubectl apply -f` them.
```

Comment:
What if we change `we` to `you`, mention kubectl and kustomize (as both install options are listed in the docs now) and also mention that if we happen to remove a resource (while not frequent at all, it might happen) the user needs to take care of it manually.
```suggestion
After that, usually you can install the new metallb version by just downloading the new manifests and either use `kubectl` or kustomize to apply them. In rare cases, you will need to manually remove a kubernetes resource if it is not present in the new manifests (e.g. if a resource `kind: ClusterRole` is removed because the permission is not needed anymore).
```

Reply:
LGTM